### PR TITLE
update sidebar

### LINKED ISSUE #213: Update Sidebar Icons
Update the Sidebar Icons for something more representative and visually appealing.

image

### PR DESCRIPTION
**Pull request by adriexnet**
### What does this PR do?

- Resolve #213

### Steps to test

1. make run lacchain 
2. navigate to http://localhost:3000/entities

#### CheckList

- [x] Follow proper Markdown format
- [x] The content is adequate
- [x] The content is available in both english and spanish
- [x] I Ran a spell check

### CODE REVIEW

**Comment by adriexnet** [comment]
@xavier506 I did the change that you recommended in slack for the footer mention, it probably resolve #224

File: webapp/src/components/Sidebar.js [hunk 1]
```diff
@@ -250,14 +268,16 @@ SidebarCategory.propTypes = {
 }
 
 const SidebarLink = ({ name, icon, to, badge }) => {
+  console.log('=>', to)
```
Comment (xavier506) [inline webapp/src/components/Sidebar.js:271]: do we need the console log?

Comment (adriexnet) [reply webapp/src/components/Sidebar.js]: My mistake I forget remove it, it's fixed in my last commit

File: webapp/src/config/general.js [hunk 2]
```diff
@@ -5,3 +5,5 @@ export const eosRateLink = process.env.REACT_APP_EOS_RATE_LINK
 export const defaultProducerLogo =
   process.env.REACT_APP_DEFAULT_PRODUCER_LOGO ||
   'https://bloks.io/img/eosio.png'
+export const appVersion = process.env.REACT_APP_VERSION || 'v1.0'
+export const appName = process.env.REACT_APP_NAME || 'eosiodasboard'
```
Comment (xavier506) [inline webapp/src/config/general.js:9]: typo : `eosiodasboard` => `eosiodashboard `

File: webapp/src/language/en.json [hunk 3]
```diff
@@ -51,7 +51,14 @@
...
     "/help>title": "Help - Dashboard",
-    "/help>heading": "Help"
+    "/help>heading": "Help",
+    "networkInformation": "Network Information",
+    "tools": "Tools",
+    "docs": "Docs",
+    "community": "Community",
+    "changelog": "Changelog",
+    "github": "Github Repository",
+    "telegram": "Telegram Room"
```
Comment (xavier506) [inline webapp/src/language/en.json:61]: Telegram Channel

File: webapp/src/config/general.js (hunk 2 above)
Comment (adriexnet) [reply webapp/src/config/general.js]: nice catch, fixed in my last commit

File: webapp/src/language/en.json (hunk 3 above)
Comment (adriexnet) [reply webapp/src/language/en.json]: fixed in my last commit

**Pull request merged**